lib/hammer_cli/output/definition.rb

```diff
@@ -54,8 +54,84 @@ def empty?
       @fields.empty?
     end
 
+    def field_sets
+      nested_fields_sets(@fields).uniq.sort
+    end
+
+    def sets_table
+      fields_col_size = average_label_length
```

Comment:
What was the reason for the average length here? It tends to produce unnecessarily narrow tables with part of the text hidden behind the dots.
```
  +------------+-----+---------+------+
  |   Fields   | ALL | DEFAULT | THIN |
  +------------+-----+---------+------+
  | Id         |  x  |    x    |  x   |
  | Name       |  x  |    x    |  x   |
  | Operati... |  x  |    x    |      |
  | Host Group |  x  |    x    |      |
  | IP         |  x  |    x    |      |
  | MAC        |  x  |    x    |      |
  | Global ... |  x  |    x    |      |
  | Organiz... |  x  |         |      |
  | Location   |  x  |         |      |
  | Additio... |  x  |         |      |
  | Content... |  x  |    x    |      |
  | Lifecyc... |  x  |    x    |      |
  +------------+-----+---------+------+
```
Wouldn't it be better to take terminal width and fit the table in and reduce the content only when it doesn't fit?

Reply:
As per our discussion we decided to use max_label_length instead.